cvat-core/src/server-proxy.ts

```diff
@@ -2337,6 +2337,18 @@ async function socialAuthentication(): Promise<any> {
     }
 }
 
+async function selectSSOIdentityProvider(email?: string, iss?: string): Promise<string> {
+    const { backendAPI } = config;
+    try {
+        const response = await Axios.get(
+            `${backendAPI}/auth/oidc/select-idp/${(email) ? `?email=${email}` : `${(iss) ? `?iss=${iss}` : ''}`}`,
+        );
```

Comment:
```suggestion
        const response = await Axios.get(
            `${backendAPI}/auth/oidc/select-idp/`, {
                params: {...(email && {email}), ...(iss && {iss})},
            },
        );
```

This will let Axios escape the query parameters as needed.

Reply:
Agree, but I would prefer syntax with less number of implicit casts, like:

params: {...(email ? {email} : {}), ...(iss ? {iss} : {})},

Reply:
By implicit casts I mean this: ``{ false } => {}``